gef.py

```diff
@@ -3278,7 +3278,28 @@ def get_keystone_arch(arch=None, mode=None, endian=None, to_string=False):
     keystone = sys.modules["keystone"]
     if (arch, mode, endian) == (None, None, None):
         return get_generic_running_arch(keystone, "KS", to_string)
-    return get_generic_arch(keystone, "KS", arch, mode, endian, to_string)
+
+    if arch in ["ARM64", "SYSTEMZ"]:
+        modes = [None]
+    elif arch == "ARM" and mode == "ARMV8":
+        modes = ["ARM", "V8"]
+    elif arch == "ARM" and mode == "THUMBV8":
+        modes = ["THUMB", "V8"]
```

Comment:
yes, it does. It can be seen in the `kstool` from keystone. To check you can try this:
```python
from keystone import *
ks1 = Ks(KS_ARCH_ARM, KS_MODE_ARM|KS_MODE_V8)
ks2 = Ks(KS_ARCH_ARM, KS_MODE_THUMB|KS_MODE_V8)
```